backend/typescript/prisma/schema.prisma

```diff
@@ -35,6 +36,17 @@ model serviceRequest {
   requestType   ServiceRequestType
 }
 
+model volunteerSignUp {
+  id        String @id @default(auto()) @map("_id") @db.ObjectId
+  v         Int    @map("__v")
```

Comment:
Looks good to go, small thing - can you take out this v - we don't need it.

Reply:
Done!